test/simulator/main/org/apache/cassandra/simulator/ClusterSimulation.java

```
@@ -919,6 +920,9 @@ public void afterStartup(IInstance i)
                                              Choices.random(random, builder.consensusChanges),
                                              minRf, initialRf, maxRf, null);
         this.factory = factory;
+
+        // during cluster shutdown ignore all failures as there is no reason to track them
+        onPreShutdown.add(() -> {simulated.failures.ignoreFailures(); return null;});
```

this is similar to what we do in jvm-dtest; when we shutdown we ignore errors.  I was seeing that `Failures` was holding on to every interrupt which made things harder to debug.  It shouldn't be checked anymore at that point so doesn't really have an impact; but in debugger it was harder to see what real issues existed